libtsuba/include/tsuba/tsuba.h

```diff
@@ -53,11 +54,11 @@ OpenFlagsValid(uint32_t flags) {
   return (flags & ~(kReadOnly | kReadWrite)) == 0;
 }
 
-KATANA_EXPORT katana::Result<RDGHandle> Open(
-    const std::string& rdg_name, uint32_t flags);
+KATANA_EXPORT katana::Result<RDGManifest> FindManifest(
+    const std::string& rdg_name);
 
 KATANA_EXPORT katana::Result<RDGHandle> Open(
-    const std::string& rdg_name, uint64_t version, uint32_t flags);
+    const RDGManifest rdg_manifest, uint32_t flags);
```

Comment:
`const` with no `&` should probably be a const reference  no?

Reply:
My bad. Should be without const. Good catch!